src/parser/jobs/pld/modules/FightOrFlight.tsx

```diff
@@ -173,13 +166,20 @@ export default class FightOrFlight extends Module {
 		}
 	}
 
-	private onRemoveFightOrFlight() {
-		if (!this.fofState.isRushed) {
+	private onRemoveFightOrFlight(event: BuffEvent) {
+		// If the enemy is untargetable at the end of FoF, the player was rushed - forgive missed hits.
+		// While technically this will never be called beyond the end of the fight, let's be really sure
+		const wasRushed = event.timestamp >= this.parser.eventTimeOffset + this.parser.pull.duration
+			|| this.invuln.isInvulnerable('all', event.timestamp)
+			|| this.invuln.isUntargetable('all', event.timestamp)
+
+		if (!wasRushed) {
```

Comment:
this module was storing the `goringTooClose` straight in the `errorResult`.
given it considers both pre-downtime _and_ pre-end-of-fight as rush, it's safe to wait for the removal of the buff, when rush is checked, and check then, rather than the predicted duration in advance